package.json

```diff
@@ -37,9 +37,7 @@
   "dependencies": {
     "class-autobind": "^0.1.0",
     "classnames": "^2.0.0",
-    "lodash.inrange": "^3.3.6",
-    "lodash.merge": "^4.6.0",
-    "lodash.nth": "^4.11.2",
+    "lodash": "^4.17.11",
```

Comment:
Using [bundlephobia](https://bundlephobia.com/) to generate some bundle size diffs make me wary of this change


- `lodash.nth@4.11.2` - 1.1kb
- `lodash.merge@4.6.0` - 11.2kb
- `lodash.inrange@3.3.6` - .9kb
- `lodash@4.17.11` - **68.5kb**

Reply:
Agreed - we were hoping to use this to optimize our lodash bundling, but given that tree shaking doesn't work with vanilla lodash, and importing modules in this way pulls in the entire lodash package, we will revert this and publish a patch release